### PR TITLE
[v6r13] Fix pilot cpu time - urgent

### PR DESCRIPTION
**Pull request by fstagni**
Since... ever we've been passing to the JobAgent CPU time variables that are unused. Also the initial default used made little sense.

### CODE REVIEW

**Comment by fstagni** [comment]
This should probably be merged even on v6r12

**Comment by phicharp** [comment]
Review OK

**Comment by fstagni** [comment]
@atsareg this is really an important one, please proceed with merging it.

File: Resources/Computing/ComputingElement.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def getResourceDict( ceName = None ):
   # now add some defaults
   resourceDict['Setup'] = gConfig.getValue( '/DIRAC/Setup', 'None' )
   if not 'CPUTime' in resourceDict:
-    resourceDict['CPUTime'] = maxCPUSegments[-1]
+    resourceDict['CPUTime'] = gConfig.getValue( '/LocalSite/CPUTime' )
```
Comment (atsareg) [inline Resources/Computing/ComputingElement.py]: Shoud the first value be the default one ? 
resourceDict['CPUTime'] = gConfig.getValue( '/LocalSite/CPUTime',  maxCPUSegments[-1])

The CE can be used not only in the local site context

Comment (fstagni) [inline Resources/Computing/ComputingElement.py]: I personally find it debatable the need for a hardcoded default. 
But I agree that for a patch it's better to play it safe, so I'll push this change in a second.

**Pull request merged**